MacOSExample/macos/Podfile

```diff
@@ -5,23 +5,33 @@ prepare_react_native_project!
 
 target 'MacOSExample-macOS' do
   platform :macos, '10.15'
-  use_native_modules!
+  
+  react_native_path = "../node_modules/react-native-macos"
+  
+  config = use_native_modules!
 
-  # Flags change depending on the env values.
-  flags = get_default_flags()
+  # Override default react-native path for MacOS building
+  config[:reactNativePath] = react_native_path
 
   use_react_native!(
-    :path => '../node_modules/react-native-macos',
-    :hermes_enabled => false,
+    :path => react_native_path,
+    :hermes_enabled => true,
```

Comment:
Do we have to change it to `true`?

Reply:
No, changed to `false` in [46a46ac](https://github.com/software-mansion/react-native-gesture-handler/pull/3055/commits/46a46ac95ce6fa6d33334c362cea374a0e972d6a)

Reply:
Actually, can we keep hermes? It's been the default for quite some time and Reanimated has intentional memory leaks on JSC to prevent crashes IIRC.

(I tried to switch it locally and the `loading bundle` overlay never disappeared for me, I'm not sure whether it was a local issue or not)

Reply:
>I tried to switch it locally and the loading bundle overlay never disappeared for me, I'm not sure whether it was a local issue or not

Occurs on less than 1/10 runs for unknown reason. App re-run fixes this issue.